pkgs/development/compilers/sbcl/2.x.nix

```diff
@@ -80,8 +81,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ texinfo ];
   buildInputs = lib.optionals coreCompression [ zstd ];
 
-  # There are no patches necessary for the currently enabled versions, but this
-  # code is left in place for the next potential patch.
+  patches = lib.optionals (stdenv.hostPlatform.system == "aarch64-darwin" && version == "2.4.0") [
```

Comment:
Does it need to be conditional on the platform? Sure, it change Darwin-only code, but it won't break anything for Linux and the condition would be simpler.

Reply:
Fair enough